src/types/num.rs

```diff
@@ -216,17 +161,141 @@ int_convert_u64_or_i64!(
     ffi::PyLong_AsUnsignedLongLong
 );
 
-#[cfg(not(Py_LIMITED_API))]
-int_convert_128!(i128, 16, 1);
-#[cfg(not(Py_LIMITED_API))]
-int_convert_128!(u128, 16, 0);
+// manual implementation for 128bit integers
+#[cfg(not(any(Py_LIMITED_API, PyPy)))]
+mod int128_conversion {
```

Comment:
What is your intention of this change?

Reply:
Without grouping in a `mod`, I had to add many `#[cfg(not(PyPy)]` annotations (e.g. also for all the tests).

As this mod is private and only contains trait implementations, I thought it would keep our code more tidy to group in a module with a single top-level `#[cfg]`.